### PR TITLE
chore: onboard out of band security

### PR DESCRIPTION
**Pull request by Saipriyavk**
Added terraform-google-out-of-band-security to list of CFT modules.

### CODE REVIEW

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -644,5 +644,10 @@ locals {
       owners      = ["xsxm", "ivanmkc", "balajismaniam", "donmccasland"]
       groups      = ["dee-data-ai", local.jss_common_group]
     },
+    {
+      name            = "terraform-google-out-of-band-security"
```
[the remark this answers is not in the document]
Comment (bharathkkb) [reply infra/terraform/test-org/org/locals.tf:648]: Could also add ` topics      = local.common_topics.net` so this is considered a networking blueprint.

**Review by bharathkkb — COMMENTED** [review]
Thanks for the PR @Saipriyavk. Please run a `terraform fmt` so the linter is happy.

File: infra/terraform/test-org/org/locals.tf (hunk 1 above)
[the remark this answers is not in the document]
Comment (Saipriyavk) [reply infra/terraform/test-org/org/locals.tf:648]: Done. Thanks!

**Comment by bharathkkb** [comment]
this has been applied

**Pull request merged**